projects/onto-search/src/lib/onto-search-facet/dropdown-multi-select-facet/dropdown-multi-select-facet.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="dropdown-multi-select" appCypressData="dropdown-multi-select">
+  <div appCypressData="checkbox-facet-title">
```

Comment:
`checkbox-facet-title`?

Reply:
Same applies to facet class names. We must have one common class for all facets e.g. `search-facet` and in addition comes the class with the concrete facet name.